main/inc/ajax/course_home.ajax.php

```diff
@@ -99,6 +99,7 @@
             $requestedVisible = $visibility == 0 ? 1 : 0;
             $requested_view = $visibility == 0 ? 'visible.png' : 'invisible.png';
             $requestedVisible = $visibility == 0 ? 1 : 0;
+            $requested_fa_class = $visibility == 0 ? 'fa fa-eye '.$requested_class : 'fa fa-eye-slash '.$requested_class;
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 121 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5714830)